src/providers/microsoft/azure/mock_tests.rs

```diff
@@ -234,6 +266,8 @@ fn test_vmsize() {
     let r = attributes.get("AZURE_VMSIZE");
```

Comment:
Let's add a check here for an attribute which comes from the shared config, e.g. `AZURE_IPV4_VIRTUAL`?

Reply:
Done